build_scripts/windows/scripts/build.cmd

```diff
@@ -187,7 +187,7 @@ msbuild /t:rebuild /p:Configuration=Release %REPO_ROOT%\build_scripts\windows\az
 
 if %errorlevel% neq 0 goto ERROR
 
-start %OUTPUT_DIR%
+echo %OUTPUT_DIR%
```

Comment:
Using start command in 1ES agent pool gets stuck, replace with echo command.
![image](https://user-images.githubusercontent.com/18628534/195894761-d2ef3f8b-6475-4c49-adcf-11770fcd8e07.png)
![image](https://user-images.githubusercontent.com/18628534/195894817-667f7f07-b53e-455a-8dad-261dedb28f89.png)